src/clj_momo/lib/clj_time/core.cljc

```diff
@@ -0,0 +1,595 @@
+(ns clj-momo.lib.clj-time.core
+  (:refer-clojure :exclude [= extend second])
+  #?(:cljs (:require-macros [clj-momo.lib.clj-time.macros :refer [immigrate]]))
+  (:require [clj-momo.lib.clj-time.coerce :as time-coerce]
+            #?(:clj  [clj-momo.lib.clj-time.macros :refer [immigrate]])
+            #?(:clj  [clj-time.core :as time-delegate]
+               :cljs [cljs-time.core :as time-delegate]))
+  #?(:clj
+     (import [java.util Calendar Date GregorianCalendar TimeZone]
+             [org.joda.time ReadablePartial ReadableInstant ReadablePeriod
+              DateTime DateTimeConstants DateTimeZone Period])
+     :cljs
+     (:import goog.date.Date
+              goog.date.DateTime
+              goog.date.UtcDateTime)))
+
+(def =
+  #?(:clj  clojure.core/=
+     :cljs time-delegate/=))
+
+(immigrate time-delegate
+           [abuts?
+            days
+            days?
+            end
+            extend
+            hours
+            hours?
+            interval
+            local-date
+            local-date-time
+            millis
+            mins-ago
+            minutes
+            minutes?
+            months
+            months?
+            now
+            number-of-days-in-the-month
+            overlap
+            overlaps?
+            seconds
+            seconds?
+            start
+            time-now
+            today
+            today-at
+            utc
+            weeks
+            weeks?
+            years
+            years?])
+
+(defprotocol DateTimeProtocol
```

Comment:
`DateTimeProtocol` is already defined in `clj-time` and `cljs-time`. Why is it necessary to define a new protocol here ? You could just extends the existing protocols for `java.util.Date` and `js/Date`.